docs/connector-development/connector-builder-ui/incremental-sync.md

```diff
@@ -92,11 +90,13 @@ In some cases, it's helpful to reference the start and end date of the interval
 
 The description above is sufficient for a lot of APIs. However there are some more subtle configurations which sometimes become relevant. 
 
-### Step
+### Split up interval
```

Comment:
The Tutorial docs should also be updated to reflect the changes here. Current things I'm seeing that should be changed:
- Rename `Datetime format` to `Cursor Field Datetime Format` so it matches the UI label
- Move the `Set the "Cursor granularity"` step  next to the step directing the user to set `Step` in Advanced and make sure to maention that they are both under `Split up interval` toggle